plugins/scaffolder/api-report.md

```diff
@@ -229,6 +230,8 @@ export const TextValuePicker: ({
   rawErrors,
   formData,
   uiSchema: { 'ui:autofocus': autoFocus },
+  idSchema,
+  placeholder,
 }: FieldProps<string>) => JSX.Element;
```

Comment:
Strange that this changes the API, but maybe because of possible default values.